### PR TITLE
Get closer to Search working. Tapping on items still doesn't work.

### PR DESCRIPTION
**Pull request by stephenjust**
I needed to change the serializers for this to work, which required clearing the ES database.

### CODE REVIEW

File: QuestionApp/src/ca/ualberta/cs/cmput301f14t14/questionapp/view/GenericSearchItemAdapter.java [hunk 1]
```diff
@@ -38,9 +38,10 @@ public View getView(int position, View convertView, ViewGroup parent){
 		TextView gDate = (TextView) convertView.findViewById(R.id.generic_date);
 		gType.setText(item.getType());
 		gText.setText(item.getBody());
-		gLocation.setText(item.getLocation().toString());
+		if (item.getLocation() != null)
+			gLocation.setText(item.getLocation().toString());
 		gDate.setText(item.getDate().toString());
-		return parent;
+		return convertView;
```
Comment (sdwowk) [inline QuestionApp/src/ca/ualberta/cs/cmput301f14t14/questionapp/view/GenericSearchItemAdapter.java:45]: haha oops I forgot to change parent when i created this....

**Comment by sdwowk** [comment]
looks good!

**Comment by RyanKusters** [comment]
I think for Answers and Comments your are missing the set parent attributes.

**Comment by stephenjust** [comment]
@RyanKusters - Those are unrelated. I'll look at those separately. This needs to get through soon.

**Comment by lamorie** [comment]
Where does it currently crash with this implemented?

**Comment by stephenjust** [comment]
No crash after this pull request (though I'll have to clear ES again,
because people are still using the old version)

## 

Stephen Just

**Comment by Cotto4** [comment]
Then then then....merge it! Merge it I say! Looks fine to me.

**Pull request merged**